.circleci/config.yml

```diff
@@ -24,12 +24,6 @@ jobs:
             - ruby-cache-{{ arch }}-{{ .Branch }}-
             - ruby-cache-
 
-      - run:
-          name: Setup Code Climate test-reporter
-          command: |
-            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 -o ./cc-test-reporter
-            chmod +x ./cc-test-reporter
```

Comment:
This may still report coverage as this is an open source project.

Reply:
Does coverage reporting not require a CodeClimate account?

Reply:
I believe that it's free for open source: https://codeclimate.com/github/ezcater/sidekiq_publisher